backend/build_scripts/update_lambda_layers.py

```diff
@@ -29,6 +29,9 @@ def get_layers(config: dict) -> list:
     for layer in config.get("Layers", []):
         print(layer["Arn"])
         arn = layer["Arn"].rsplit(":", maxsplit=1)[0]
+        if "Datadog-Extension" in arn or "Datadog-Python" in arn:
```

Comment:
What's this for?

Reply:
Some lambdas have been instrumented to report metrics to Datadog. 
These lambdas have 2 datadog layers that should not be updated.
Custom Instrumentation Doc: https://docs.datadoghq.com/serverless/aws_lambda/installation/python/?tab=custom


I'll take this out and move it to the upcoming Datadog PR: https://github.com/WarnerMedia/artemis/pull/240